docs/user/security/rbac_tutorial.asciidoc

```diff
@@ -0,0 +1,119 @@
+[[space-rbac-tutorial]]
+=== Tutorial:  Use Role Based Access Control (RBAC) to customize Kibana Spaces
+
+Role Based Access Control (RBAC) is used to provide access to data, tools,
+and Kibana Spaces.  In this tutorial you will learn how to configure roles
+that provide the right users with the right access to the data, tools, and
+Kibana Spaces.
+
+[float]
+==== Scenario
+
+Our user is a web developer working on a bank's
+online mortgage service.  The web developer has these 
```

Comment:
Suggest:

The web developer requires access to the following:

- The data for the service
- Tools to visualize the data and create a dashboard
- Tools to monitor the performance of the system

Reply:
I think the existing text gives the reader a realistic set of requirements for a user that many of our customers will have experience with.